src/cli/init.rs

```diff
@@ -70,28 +94,63 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     // Fail silently if it already exists or cannot be created.
     fs::create_dir_all(&dir).ok();
 
-    // Write pixi.toml
-    let name = dir
-        .file_name()
-        .ok_or_else(|| {
-            miette::miette!(
-                "Cannot get file or directory name from the path: {}",
-                dir.to_string_lossy()
-            )
-        })?
-        .to_string_lossy();
-    let version = "0.1.0";
-    let author = get_default_author();
-    let channels = if let Some(channels) = args.channels {
-        channels
+    let (name, channels, conda_deps, pip_deps) = if let Some(env_file) = args.env_file {
+        let env_info = read_env_yml(env_file)?;
+        let name = env_info.name;
+        let channels = env_info.channels;
+
+        let (mut conda_deps, mut pip_deps) = (vec![], vec![]);
```

Comment:
Haha this is very pytonic. Just write:

```suggestion
        let mut conda_deps = vec![];
        let mut pip_deps = vec![];
```